docs/source/using/addons.adoc

```diff
@@ -28,13 +28,18 @@ The following example shows the contents of an add-on file, including the name a
 # Required-Vars: ACME_TOKEN                                                             // <3>
 # OpenShift-Version: >3.6.0                                                             // <4>
 
-oc adm policy add-scc-to-group anyuid system:authenticated                              // <5>
+mytoken := oc sa get-token oauth-client                                                 // <5>
+
+oc new-app -p OPENSHIFT_OAUTH_CLIENT_SECRET=#{mytoken}                                  // <6>
+oc adm policy add-scc-to-group anyuid system:authenticated                              // <7>
 ----
 <1> (Required) Name of the add-on.
 <2> (Required) Description of the add-on.
 <3> (Optional) Comma separated list of required interpolation variables. See xref:../using/addons.adoc#addon-variable-interpolation[Variable Interpolation]
 <4> (Optional) OpenShift version required to run a specific add-on. See xref:../using/addons.adoc#addon-openshift-version-semantics[OpenShift Version Semantic]
-<5> Actual add-on command. In this case, the command executes the *oc* binary.
+<5> Run an actual add-on command and store its output in a variable.
+<6> Actual add-on command. In this case, the command executes the *oc* binary using the `mytoken` variable value.
+<7> Actual add-on command. In this case, the command executes the *oc* binary.
```

Comment:
@praveenkumar I think we need a section for documenting around storing an command output and reusing it . I would suggest section name as `Internal variables` or something better. The reason behind a new section is because we mostly have sections for each add-on features, so unless we create an section users might miss to notice the feature.

Reply:
```
=== Internal Variables

Add-on allows to define variables in the add-on. You can save a command out put in a variable and use the variable in subsequent add-on commands example :
----
mytoken := oc sa get-token oauth-client 
oc new-app -p OPENSHIFT_OAUTH_CLIENT_SECRET=#{mytoken}
----